resources/js/utilities/formatDate.ts

```diff
@@ -0,0 +1,36 @@
+export function formatFullDateTime(
```

Comment:
Do you need to export that function? I see you only use it inside another exported function. 

```suggestion
function formatFullDateTime(
```